Source/NSURLProtocol.m

```diff
@@ -847,7 +847,6 @@ - (void) startLoading
 	self, @"HEAD",
 	self, @"GET",
 	self, @"POST",
-	self, @"PATCH",
```

Comment:
Why did you remove PATCH here?

Reply:
I noticed this after I pushed.  I am not sure how this happened, as I branched from the testplant branch.  I can manually merge the branch to ensure that this does not get lost.

Reply:
I'm nervous that other changes may be lost as well?  Please be careful.